model.py

```diff
@@ -8722,147 +8804,117 @@ class CollectionSetting(Base):
          index=True, nullable=False
      ),
      UniqueConstraint('collection_id', 'licensepool_id'),
- )
+)
 
-    
-class Catalog(Base):
+collections_identifiers = Table(
+    'collections_identifiers', Base.metadata,
+    Column(
+        'collection_id', Integer, ForeignKey('collections.id'),
+        index=True, nullable=False
+    ),
+    Column(
+        'identifier_id', Integer, ForeignKey('identifiers.id'),
+        index=True, nullable=False
+    ),
+    UniqueConstraint('collection_id', 'identifier_id'),
+)
 
-    """A Catalog is like a Collection, but it doesn't hold any actual
-    LicensePools, it just records Identifiers.
 
-    The Collections associated with a Library in its circulation
-    manager will show up as Catalogs associated with the corresponding
-    Library in the metadata wrangler.
-    """
+class ClientServer(Base):
+    """A server that has authenticated access to this application.
```

Comment:
`ClientServer` is a very generic name. I would call it something like `IntegrationAccount` or `IntegrationClient`. It doesn't matter to us whether the thing making these requests is a 'server' or some other piece of software.